ocpmodels/common/utils.py

```diff
@@ -545,11 +545,24 @@ def get_pbc_distances(
 
 
 def radius_graph_pbc(
-    data, radius, max_num_neighbors_threshold, pbc=[True, True, False]
+    data, radius, max_num_neighbors_threshold, pbc=[True, True, True]
 ):
     device = data.pos.device
     batch_size = len(data.natoms)
 
+    if hasattr(data, "pbc"):
+        data.pbc = np.array(data.pbc)
```

Comment:
We should prefer using torch here as numpy operation would increase GPU -> CPU and potentially be slower (although the operations here aren't too compute heavy and would be fine)